examples/ml-fraud-detection/akka-streams/src/main/scala/cloudflow/examples/frauddetection/process/ScoreTransactionsAndServeModels.scala

```diff
@@ -0,0 +1,82 @@
+package cloudflow.examples.frauddetection.process
+
+import akka.Done
+import akka.pattern.ask
+import akka.stream.scaladsl.Flow
+import akka.util.Timeout
+import cloudflow.akkastream.AkkaStreamlet
+import model.{Model, ModelDescriptorUtil, MultiModelFactory}
+import model.actor.ModelServingActor
+import modelserving.model.{ModelDescriptor, ModelType}
+import cloudflow.akkastream.scaladsl.RunnableGraphStreamletLogic
+import cloudflow.streamlets.{StreamletShape, StringConfigParameter}
+import cloudflow.streamlets.avro.{AvroInlet, AvroOutlet}
+import cloudflow.examples.frauddetection.data.{CustomerTransaction, ScoreFromTheModel, ScoredTransaction}
+import cloudflow.examples.frauddetection.models.tensorflow.{FraudTensorFlowBundledModelFactory, FraudTensorFlowModelFactory}
+
+import scala.concurrent.duration._
+
+final case object ScoreTransactionsAndServeModels extends AkkaStreamlet {
+
+  //\\//\\//\\ INLETS //\\//\\//\\
+  val transactionsComeInHere = AvroInlet[CustomerTransaction]("transactions")
+
+  //\\//\\//\\ OUTLETS //\\//\\//\\
+  val scoredTransactionsComeOutHere = AvroOutlet[ScoredTransaction]("results")
+
+  //\\//\\//\\ SHAPE //\\//\\//\\
+  final override val shape = StreamletShape.withInlets(transactionsComeInHere).withOutlets(scoredTransactionsComeOutHere)
+
+  val modelFactory = MultiModelFactory(
```

Comment:
I think we can pair this down a little further, please have a look at tensorflow-akka example.

Reply:
+1 to Ray's suggestion, I would also suggest to add Flink (can do this once This is approved)